content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -107,10 +107,14 @@ The `--for` parameter to `istioctl waypoint apply` can be used to change the [tr
 | `waypoint-for` value | Traffic type |
 | -------------------- | ------------ |
 | `service`            | Kubernetes services |
-| `workload`           | Pod or VM IPs |
+| `workload`           | Pod IPs or VM IPs |
 | `all`                | Both service and workload traffic |
 | `none`               | No traffic (useful for testing) |
 
+{{< tip >}}
```

Comment:
Having 2 tips in the same spot seems unnecessary/duplicative?

Maybe pick one spot for the `tip` (not sure if right here, or below inline is better)

Reply:
They look much closer together in the PR than they actually are in the rendered page. Take a look here and lmk if you still think one or the other needs to go.

https://deploy-preview-15609--preliminary-istio.netlify.app/latest/docs/ambient/usage/waypoint/

Reply:
it still reads a bit as saying the same thing again a few paragraphs later, maybe just hoist the bottom tip right under the table - it makes more sense _to me_ to explicitly call this out once under the types than at the end of the section, but that's just an opinion, so feel free to ignore.